lib/model/autocomplete.dart

```diff
@@ -244,6 +234,40 @@ class MentionAutocompleteView extends ChangeNotifier {
     notifyListeners();
   }
 
+  Future<List<R>?> _computeResults(Q query);
```

Comment:
This could be refactored to avoid duplications by using a sort of strategy pattern.